test/src/test/java/org/corfudb/runtime/view/stream/StreamAddressSpaceTest.java

```diff
@@ -175,6 +181,47 @@ public void testStreamSerialization() throws Exception {
         assertThat(obj1.toArray()).isEqualTo(deserialized.toArray());
     }
 
+    @Test
```

Comment:
Should we add a test for the forEachUpTo?

Reply:
Done.